### DISCUSSION TITLE
Logo di ODS

### DISCUSSION THREAD

**Discussion by cirospat** [opendatasicilia/opendatasicilia.it#33] [Generale]
Buonasera a tutti/e! 
Apro questa discussion per raccogliere le posizioni e le idee dei membri della community sul  **`logo`** di ODS.

Questa Discussion (**Logo di ODS**) parte dalla [**Discussion sul co-design**](https://github.com/opendatasicilia/opendatasicilia.it/discussions/28).

La community di ODS utilizza un logo da anni (circa 6-7 anni). 
![image](https://user-images.githubusercontent.com/3757525/145685263-46cd4121-23ab-456f-a621-1a453a88225a.png)

Il logo è un icona/immagine che veicola, soprattutto online, il nome/brand di un progetto, di una community, di un azienda, ecc.

Possiamo confrontarci qui, per capire:

- chi è per cambiarlo e quali sono le ragioni del cambiamento, e quali  potrebbero essere le proposte grafiche (da postare nei commenti);
- chi è per mantenere l'attuale logo e quali sono le ragioni del mantenimento.

Importante è capire (per tutte/i) le ragioni di entrambe le posizioni. 

Buon confronto

**Comment by gbvitrano** [comment]
Non vedo il motivo per cambiare/modificare un logo che ha fatto la storia di ODS.
- Funziona bene nel web come immagine e icona
- Si legge bene nella carta stampata sia a colori che in B/N,
- E' perfetto per gadget



![photo5882030267210774460](https://user-images.githubusercontent.com/20488693/145686324-9b20cc00-f203-4f40-8aad-8b337055fc51.jpg)
![photo5882030267210774459_00](https://user-images.githubusercontent.com/20488693/145686316-9f510c19-53c1-482a-8513-f5d5f8683083.jpg)
![photo5882030267210774461](https://user-images.githubusercontent.com/20488693/145686321-1a84da4f-845b-49a3-b536-f6f014caba0b.jpg)
![photo5882030267210774458](https://user-images.githubusercontent.com/20488693/145686339-6d7d50cf-7e8f-4a57-b324-e4b0a3643198.jpg)

**Reply by dennisangemi** [reply]
Ciao @gbvitrano la nuvoletta dalla quale piove codice binario da dove proviene? L'ha disegnata uno/a di noi? contraddistingue solo noi? A cosa si riferisce di preciso?

**Comment by dennisangemi** [comment]
Buonasera a tutti/e! Nella discussion [co-design](https://github.com/opendatasicilia/opendatasicilia.it/discussions/28) sono stati inseriti tanti commenti relativi all'eventualità di cambiare logo per via del protipo presentato ma mi sono reso conto del fatto che quella di presentare il prototipo con un nuovo logo non è stata la migliore delle idee. Utilizzo questa discussion per raccogliere dei parerei relativi al logo così da mantenere un ordine tra i topics.

### Perchè cambiare logo?
Premesso che apprezzo molto l'attuale logo, di seguito inserisco alcuni punti che mi hanno portato a pensare all'eventualità di cambiarlo. Sono tutti pareri personali (e quindi potrebbero essere non condivisibili) e alcune domande le cui risposte chiaramente potrebbero farmi cambiare idea:
- Mi dà l'impressione di qualcosa di datato a cui serverebbe una "svecchiata"
- Secondo me racconta poco di noi perchè contiene pochi elementi che rimandano a ciò che siamo e ciò che facciamo: l'unico elemento (molto carino) con un fine specifico secondo me è la nuvoletta dalla quale piove codice binario che mi fa pensare al cloud, alla programmazione. Questo è sufficiente a identificarci? 
- C'è un motivo particolare per il quale è presente un esagono? Il 6 è un numero importante per noi? (Es. magari ha senso perchè ODS nasce nel 2006 o perchè i "fondatori" sono 6. Non so)
- Il nome della nostra community riportato nel logo è `opendata <br> sicilia` e secondo me questo determina confusione in chi deve citarci. Ho visto articoli in cui veniamo chiamati nei modi più disparati: opendatasicilia, opendata sicilia, open data sicilia, open data Sicilia, OpenDataSicilia, Open Data Sicilia, #opendatasicilia. Secondo me è necessario definire meglio la nostra identità da questo punto di vista in modo tale da avere una "denominazione univoca", un solo nome.

### La mia proposta
![image](https://user-images.githubusercontent.com/77018886/145685561-1dc94bef-f23f-4b63-80e2-cc7d7a6a6fc4.png)

Ancora è solo un'idea, non sono nemmeno sicuro piaccia a me però secondo me è un punto di partenza. Ho cercato di dare un senso ad ogni elemento. Di seguito alcune riflessioni:
- L'ispirazione proviene essenzialmente dalle celle di un foglio di calcolo. Quando penso ai "dati", nella mia testa compare immediatemente l'immagine di una tabella. 
- **Banalmente** (forse fin troppo) non tutte le celle sono _chiuse_ perchè vogliamo che i dati siano _open_ ;)
- I bordi delle celle rivestono un duplice ruolo: descrivere i limiti delle celle e collegare i _pallocchi_ 
- Cosa sono i _pallocchi_? Sono dati ma anche membri di una comminuty, connessi perchè condividono uno _scopo comune_ (quello indicato nel sito: far conoscere e diffondere la cultura dell’open government e le prassi dell'open data)

È solo una proposta. Forse sarà la prima proposta (in vista di altre) o forse rimarrà una proposta. Non lo so ancora e dipende più che altro dal tempo disponibile.

### Alcuni commenti
> @pigreco on Discussion #28 
> sono particolarmente legato a questo logo di OpenDataSicilia e mi piacerebbe molto restasse questo.

> @cirospat on Discussion #28 
> a me piace l'attuale logo. Perché?
> Lo abbiamo vissuto con passione e sentimento. Scola sudore. Ci identifica in tutta Italia.
> Non lo vedo come qualcosa da cambiare.
> Questo il mio umile pensiero.

> @gbvitrano on Discussion #28 
> Restyling e Rebranding sono cose molto diverse da non confondere.
> Sentiti libero di proporre tutto ciò che vuoi, ma il logo di ODS per me in questo momento ed in questa fase non è in discussione. Quel logo che tu vuoi cambiare... è la storia di ODS

**Comment by gabacode** [comment]
Sinceramente mi piacciono entrambi, la valenza storica e tradizionale del logo attuale non è da sottovalutare, perchè è il simbolo di ODS, e della gente che ne fa parte. 

Mi piace la nuvoletta, mi comunica qualcosa come "piovono dati" e come la pioggia, nutrono la terra e sono fonte di vita. L'esagono mi ricorda quello di un alveare, che comunica creazione, e il fatto che si "incastra" insieme ad altri esagoni.

Si potrebbe forse associare al logo, anche un logotype alternativo, che sia complementare al logo già esistente, in situazioni come la recente proposta landing page? Quindi essenzialmente usarli entrambi a seconda della necessità?

Le argomentazioni di Dennis hanno molto senso e significato dietro le scelte stilistiche del nuovo logo proposto, e questa è una cosa molto importante. Uniformare la dicitura dell'associazione è anche importante a livello di SEO e riferimenti vari, io di solito riferisco come OpenDataSicilia, è stato mai definito uno standard a riguardo? I pallocchi e l'apertura dei dati sono anch'essi elementi molto validi, e secondo me dovrebbero rifletterne l'intento all'interno del logo/logotype.

Si potrebbe anche provare a fare qualche modifica partendo dal logo originale. Un esempio usando i pallocchi:

![Untitled-1](https://user-images.githubusercontent.com/38985346/145687168-469845ce-d8a2-474e-9512-bc8eef693b20.png)

p.s. @gbvitrano adesso che le ho viste vorrei troppo una maglietta di ODS :D

**Comment by gbvitrano** [comment]
Così `opendata <br> sicilia` non fuziona  e  `open <br> data <br> sicilia`  funziona? :-) ho qualche dubbio, ma io di design non ci capisco nulla :-) ti consiglio di dare un'occhiata ai lavori di Bob Noorda qualche logo l'ha **svecchiato** :-) :-) 

@dennisangemi il logo deve funzionare a qualsiasi scala e su qualsiasi materiale, esempio... se riduci le dimensioni ad icona,  i pallocchi e linee li perdi o potrebbero dare fastidio ad atri elementi del logo. 
In questa fase è più importante studiare bene il sito, per il logo ci sarà tempo per **STUDIARE** un restyling

**Reply by dennisangemi** [reply]
@gbvitrano anche io non sono un graphic designer, sono solo un po' appassionato. Riconosco di non essere all'altezza di realizzare un logo degno di questo nome ma mi piace sperimentare.

Per quanto riguarda il nome della community, la mia proposta non migliora la situazione, hai ragione. 

Se riduci a icona la mia proposta, perdi i pallocchi, è vero; se riduci a icona l'attuale logo, perdi "opendata sicilia" (ed è una cosa che ho sempre notato ma ho dimenticato di inserire nel mio commento precedente)

**Reply by gbvitrano** [reply]
ma l'attuale logo è sempre riconoscibile anche ad icona, l'esagono e la nuvoletta sono un forte segno grafico di riconoscimento.
![2021-12-11_20h11_52](https://user-images.githubusercontent.com/20488693/145688714-994aee43-3936-45bd-bf7a-9a0e28dae542.png)

Sperimenta, fai benissimo, tutto avviene per studio per processi di trasformazione, non mi sono mai innamorato della mia prima idea, metto sempre tutto in discussione, anche quando ho finito...riparto da zero :-)

**Comment by cirospat** [comment]
@dennisangemi 
> Secondo me racconta poco di noi

"`Noi`", in questa community siamo tanti. 
Siamo anche persone che diversi anni fa già organizzavamo i primi eventi di interesse nazionale, a Palermo nel 2016, a Messina, a Caltanissetta, a Pedara, a Castelbuono, a Palermo di nuovo,.....

"`Noi`" siamo persone che nel 2018 vincevamo premi nazionali: https://open.gov.it/premio-2018/premiazione-2018/
![image](https://user-images.githubusercontent.com/3757525/145686726-37f688b2-cb87-4dc8-8313-ab6a7e674296.png)

"`Noi`" siamo persone che negli anni passati abbiamo creato gadget per circondarci di quel logo nei nostri corpi (le tshirts), nei nostri smartphone e pc (adesivi) per sentirci appartenenti a questa community. 
In pratica "`Noi`" ci siamo affezionati a quel logo. Affezionati significa che ci "riconosciamo". Ci fa stare bene psicologicamente :)

"`Noi`" oggi siamo più persone di "`Noi`" degli anni passati, e questo è una fortuna per la ricchezza della comunità. C'è più diversità, anche sul logo (mantenerlo, cambiarlo). C'è crescita, c'è maggiore conoscenza. 

Ti voglio bene Dennis, perchè hai belle idee nella tua testa, ... anche se la proposta di cambiare logo non coincide con l'idea dei vecchietti :) di mantenerlo.
Ma ti voglio leggere perchè le tue ragioni sul logo mi fanno riflettere, mi fanno pensare. Mi fanno pensare alle evoluzioni che avvengono anche in una piccola società come la nostra di ODS. La proposta di ristrutturazione del sito è un evoluzione, peccato che non ci sia il bonus rifacimento sito web al contrario del bonus 110% ristrutturazione edilizia ... :) e quindi ci dobbiamo auto tassare per l'hosting.

Quel logo per noi (ora lo scrivo normale :) è stato sudore e fatica e passione nelle cose che abbiamo fatto, decine e decine di mappe e infografiche di @gbvitrano o fatte da me o da @aborruso o da altri, dove la prima cosa che inseriamo è il logo e poi costruiamo il resto.
Con quel logo abbiamo ricevuto riconoscimenti. E perchè non dirlo, ne siamo orgogliosi. Noi!

Forse i loghi e le icone dovrebbero avere minore significato nel mondo in cui viviamo, per lasciare il posto ai contenuti, ma chissà perchè  ci attacchiamo visceralmente a queste immagini/icone, abbiamo bisogno di feticci che ci rappresentano nel mondo della comunicazione soprattutto online. In fondo viviamo nel mondo dell'immagine.

Ho scritto questo solo perchè mi hai fatto ricordare di .... noi
:)

--- 

**La nuvoletta** ...... mi fa ricordare che dobbiamo stare con la testa nelle nuvole se vogliamo fare cose belle e interessanti

**Gli zeri e gli uni (il codice binario)**  ... mi fa ricordare la diversità delle persone, ricchezza pura!

**Opendata** ..... hasta la victoria siempre!

**Sicilia**  what else?

`p.s.: il preservativo con il logo di ODS non ve lo faccio vedere pubblicamente`

**Reply by dennisangemi** [reply]
Ciao @cirospat , grazie per questa riflessione. Leggendoti ho pensato a due cose:

1. faremmo bene ad inserire un po' di "storia" di ODS nella sezione "Chi siamo" del nostro sito
2. ~vogliamo vedere tutti il preservativo con il logo di ODS HAHAHHA~

**Comment by pigreco** [comment]
scusate la brevità del mio messaggio:

per me il logo è perfetto così, al massimo andrebbero "svecchiati" le persone che ci sono da troppi anni ormai :-)

ma scherzi a parte: non esiste un logo perfetto che rappresenti in tutto una comunità, quindi, perché cambiare qualcosa a cui molti decani sono affezionati?

saluti

**Comment by peppelamensa** [comment]
Ciao giovani, dico sinteticamente la mia. 
Al momento non sarei per modificare del tutto il logo, sarei per un restyling.
Se andasse bene come ipotesi, mi chiederei quali siano i suoi tratti grafici essenziali, quelli che lo fanno quello che è sempre stato, per il suo pubblico soprattutto e anche per noi, e manterrei solo quelli. 
Un restyling (che ci starebbe tutto dopo tanti anni) che lo mantenga riconoscibile, tra l'altro mentre la sua visibilità è in crescita, mi pare un passo che può tenere insieme diverse visioni. 

Andando ai tratti distintivi, io vedrei solo la nuvola con la pioggia binaria. Ma so che tirerete fuori dei lampi di genio, perciò sto pronto in ascolto 🙂

Ringrazio Dennis per i suoi input, sempre pensati, utili e freschi (anche il logo mi piace, ha i suoi perché) 🙏

**Comment by dennisangemi** [comment]
Carissimi/e, ho compreso tutte le vostre posizioni e alla fine sono d'accordo con voi: non possiamo cestinare l'attuale logo di ODS. 

Ho pensato quindi di fare una cosa del genere:
![image](https://user-images.githubusercontent.com/77018886/145732454-72635217-93c3-4304-9b3e-5a602d96e970.png)

mantenere una versione rivista dell'attuale logo che verrà utilizzato come pulsante "home" e inserire una sorta di illustrazione/logotype nella landing page (sinistra). Che ne dite?

**Reply by pigreco** [reply]
Forse nel precedente messaggio mi sono spiegato male e mi scuso, per me il logo non va modificato, non va rivisto, non va fatto nessun tipo di restyling e deve essere il protagonista nel nuovo sito e non un pulsante "home" in trasparenza, inoltre per me va tolto l'altro logo che richiamano le celle di un foglio elettronico perché potrebbe creare confusione e soprattutto c'entra poco con quello che fa ODS (non siamo una app, ODS è un modo di pensare)

Io mi identifico nel 'vecchio' logo, lo uso da anni e non vedo nessun motivo per cambiarlo o modificarlo e invito tutti a modellare il nuovo sito attorno al logo e non viceversa.

m2c

**Comment by gbvitrano** [comment]
Il doppio logo genera solo confusione, non avere fretta, studieremo con calma è serenità il logo, faremo 100.000 prove e variazioni sul tema con l'approccio suggerito da @peppelamensa.  Abbi pazienza, analizza, studia e proponi più varianti, il logo è un progetto grafico e come tutti i progetti per maturare ha bisogno di tempo :-)

**Comment by dennisangemi** [comment]
ci rinuncio

**Reply by pigreco** [reply]
@dennisangemi perché sei cosi frettoloso? 

una decisione cosi importante, sulla modifica del logo, non si può prendere in due giorni e con pochi pareri,
occorre aspettare i pareri di molte altre persone e non solo la mia; mancano i pareri di Andrea, Nino, Nelson, Davide, Giuseppe, Giovanni, Paola  ecc...

**Reply by dennisangemi** [reply]
Mi spiego meglio: ho capito  che la volontà condivisa è quella di non toccare minimamente il logo ergo per adesso ci rinuncio e mi dedico ad altro.

Scusate se sembro frettoloso ma avendo poco tempo a disposizione tendo a concentrare tutto nel weekend e le cose ancora da fare sono tante.

P.s.
Comprendo assolutamente la vostra posizione anche se non la condivido a 360°
